lang/po/new/sv.po

```diff
@@ -1,1772 +1,1758 @@
+
+# MSG_CRASH_DET_STEALTH_FORCE_OFF c=20 r=4
+msgid "WARNING:\\x0aCrash detection\\x0adisabled in\\x0aStealth mode"
+msgstr "VARNING:\\x0aKrockdetekt\\x0ainaktiverad i\\x0a tyst l\xe1ge"
+
+# MSG_BABYSTEPPING_Z c=15
+#: ultralcd.cpp:3034
+msgid "Adjusting Z:"
+msgstr "Justerar Z:"
+
+# MSG_SELFTEST_CHECK_ALLCORRECT c=20
+#: ultralcd.cpp:8490
+msgid "All correct"
+msgstr "Allt korrekt"
+
+# MSG_WIZARD_DONE c=20 r=8
+#: messages.c:118
+msgid "All is done. Happy printing!"
+msgstr "Allt \xe1r klart. God utskrift!"
+
+# MSG_AMBIENT c=14
+#: ultralcd.cpp:1727
+msgid "Ambient"
+msgstr "Omgivande"
+
+# MSG_AUTO c=6
+#: messages.c:160
+msgid "Auto"
+msgstr "Auto"
+
+# MSG_PRESS c=20 r=2
+#: ultralcd.cpp:2485
+msgid "and press the knob"
+msgstr "och tryck pa knappen"
+
+# MSG_CONFIRM_CARRIAGE_AT_THE_TOP c=20 r=2
+#: ultralcd.cpp:3355
+msgid "Are left and right Z~carriages all up?"
+msgstr "\xe1r bada Z-vagnarna helt uppe?"
+
+# MSG_AUTO_HOME c=18
+#: messages.c:11
+msgid "Auto home"
+msgstr "Auto hem"
+
+# MSG_AUTOLOAD_FILAMENT c=18
+#: ultralcd.cpp:6732
+msgid "AutoLoad filament"
+msgstr "Autoladda filament"
+
+# MSG_AUTOLOADING_ONLY_IF_FSENS_ON c=20 r=4
+#: ultralcd.cpp:4317
+msgid "Autoloading filament available only when filament sensor is turned on..."
+msgstr "Autoladdning av fil. \xe1r endast m\xefjligt n\xe1r fil. sensorn \xe1r aktiv...
+
+ MSG_AUTOLOADING_ENABLED c=20 r=4
+#: ultralcd.cpp:2648
+msgid "Autoloading filament is active, just press the knob and insert filament..."
+msgstr "Autoladdning filament \xe1r aktiv, tryck pa knappen och s\xe1tt i filament..."
+
+# MSG_SELFTEST_AXIS_LENGTH c=20
+#: ultralcd.cpp:8173
+msgid "Axis length"
+msgstr "Axell\xe1ngd"
+
+# MSG_SELFTEST_AXIS c=16
+#: ultralcd.cpp:8174
+msgid "Axis"
+msgstr "Axel"
+
+# MSG_SELFTEST_BEDHEATER c=20
+#: ultralcd.cpp:8131
+msgid "Bed/Heater"
+msgstr "B\xe1dd/V\xe1rmare"
+
+# MSG_BED_DONE c=20
+#: messages.c:15
+msgid "Bed done"
+msgstr "B\xe1dd klar"
+
+# MSG_BED_HEATING c=20
+#: messages.c:16
+msgid "Bed Heating"
+msgstr "B\xe1dden v\xe1rms upp"
+
+# MSG_BED_CORRECTION_MENU c=18
+#: ultralcd.cpp:5798
+msgid "Bed level correct"
+msgstr "B\xe1ddnivakorrigering"
+
+# MSG_BELTTEST c=18
+#: ultralcd.cpp:5778
+msgid "Belt test"
+msgstr "B\xe1lt test"
+
+# MSG_BED_LEVELING_FAILED_POINT_LOW c=20 r=6
+#: messages.c:17
+msgid "Bed leveling failed. Sensor didn't trigger. Debris on nozzle? Waiting for reset."
+msgstr "B\xe1ddnivelleringen felade. Sensorn utl\xefstes inte. Skr\xe1p pa munstycket? V\xe1ntar pa aterst\xe1llning."
+
+# MSG_BRIGHT c=6
+#: messages.c:158
+msgid "Bright"
+msgstr "Ljus"
+
+# MSG_BRIGHTNESS c=18
+#: messages.c:154
+msgid "Brightness"
+msgstr "Ljusstyrka"
+
+# MSG_BED c=13
+#: messages.c:14
+msgid "Bed"
+msgstr "B\xe1dd"
+
+# MSG_BELT_STATUS c=18
+#: messages.c:19
+msgid "Belt status"
+msgstr "B\xe1lt status"
+
+# MSG_RECOVER_PRINT c=20 r=2
+#: messages.c:82
+msgid "Blackout occurred. Recover print?"
+msgstr "Blackout intr\xe1ffat. Aterst\xe1lla utskr?"
```

Comment:
We replace the äöüß and other diacritics with the lang-import.sh script.

Please keep the se.po file as you write in Swedish with umlaut and diacritics.